reputation.php

```diff
@@ -166,6 +166,10 @@
 	{
 		$query = $db->simple_select("reputation", "*", "adduid='".$mybb->user['uid']."' AND uid='{$uid}' AND pid='0'");
 		$existing_reputation = $db->fetch_array($query);
+		if(!empty($existing_reputation['rid']))
+		{
+			$rid = $existing_reputation['rid'];
+		}
 		$rid = $existing_reputation['rid'];
```

Comment:
You have omitted to remove this line.

Reply:
Also `if($existing_reputation)` would be shorter and do the same.

Reply:
Thank you!